### PR TITLE
`toLines` on FASTA values is really wrong

### PR DESCRIPTION
**Pull request by eparejatobes**

### CODE REVIEW

**Comment by eparejatobes** [comment]
@laughedelic published

File: src/main/scala/fasta.scala [hunk 1]
```diff
@@ -62,7 +62,8 @@ case object fasta {
     @inline private def me: ValueOf[FASTA.type] = FASTA(fa)
 
     def toLines: Seq[String] =
-      Seq(s"${header.start}${me get header value}") ++ ((me get sequence value) lines)
```
Comment (laughedelic) [inline src/main/scala/fasta.scala:65]: what's up, why not lisp-style anymore? :trollface:

Comment (eparejatobes) [inline src/main/scala/fasta.scala:65]: Seq(:trollface:)

**Comment by eparejatobes** [comment]
waiting for more testing before merging

**Pull request merged**